glotaran/project/test/test_project.py

```diff
@@ -300,6 +336,31 @@ def test_create_scheme(existing_project: Project):
     assert scheme.maximum_number_function_evaluations == 1
 
 
+@pytest.mark.parametrize(
+    "data",
+    (xr.DataArray([1]), xr.Dataset({"data": xr.DataArray([1])}), "file"),
+)
+def test_create_scheme_data_lookup_overwrite(
+    tmp_path: Path, existing_project: Project, data: LoadableDataset
+):
+    """Test data_lookup_overwrite functionality."""
```

Comment:
```suggestion
    """Test data_lookup_override functionality."""
```

Reply:
Fixed in [5e7a47d](https://github.com/glotaran/pyglotaran/pull/1257/commits/5e7a47de52900b22acde581c9e934280d5e95975)